style.css

```diff
@@ -97,3 +98,225 @@ ul {
   top: 610px;
   right: 2px;
 }
+
+.recent-work {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px 120px 24px;
+}
+
+.title-work {
+  display: flex;
+  font-family: 'Crete Round', serif;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 400;
+  color: #1c1a19;
+  margin: 24px;
+  border-bottom: 1px solid #1c1a19;
+  padding-bottom: 24px;
+}
+
+.pro-img {
+  display: flex;
+  width: 100%;
+  height: auto;
+}
+
+.stories {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  align-self: flex-start;
+}
+
+.about-stories {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.cards {
+  display: grid;
+  column-gap: 12px;
+  grid-template-columns: 1fr 1fr 2fr 1fr;
+  padding-inline-start: 0;
+}
+
+.cards li {
+  border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 12px 10px 12px;
+}
+
+.link-pro {
+  text-decoration: none;
+  color: white;
+  font-size: 17px;
+  align-self: flex-start;
+}
+
+.see-pro {
+  border: 1px solid #ff6b00;
+  background: #ff6b00;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+}
+
+.second-project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(images/imgbackground.png);
+  padding: 24px;
+  margin-top: 24px;
+}
+
+.project-heading {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  color: white;
+  align-self: flex-start;
+}
+
+.description {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #fff;
+}
+
+.cards-project2 {
+  display: grid;
+  column-gap: 12px;
+  grid-template-columns: 1fr 2fr 1fr;
+  padding-inline-start: 0;
+}
+
+.cards-project2 li {
+  border: 1px solid white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 12px 10px 12px;
+  color: white;
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.link-project2 {
+  text-decoration: none;
+  color: white;
+  font-size: 17px;
+  width: 100%;
+}
+
+.see-project2 {
+  border: 1px solid #ff6b00;
+  background: #ff6b00;
+  padding: 12px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #1c1a19;
+  width: 100%;
+}
+
+.about-me-top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px 120px 24px;
+}
+
```

Comment:
- [ ] Kindly reduce or adjust the padding-bottom of this class so the button can be close to the text above it with 24px spacing.
Hint: I believe this will fix that `padding: 120px 24px 24px 24px;` instead of the style you wrote at line `242`. Please fix that so your design will look great as the one in Figma. 👍 

**See your design**
![image](https://user-images.githubusercontent.com/50941074/159893000-cdba80f6-8368-463a-847e-9ba130e3151f.png)

**The Figma design**
![image](https://user-images.githubusercontent.com/50941074/159893184-8392e838-3936-43eb-a4a1-9afb77e2aa66.png)